test-plans/template.md

```diff
@@ -0,0 +1,43 @@
+# Name
+
+Number: Unique and unambiguous identifier according to the following
+pattern:
+U.XX(service identifier).X (sequential number, start
+at 1).  
+Version: Version of the use case according to pattern.
```

Comment:
I'd simply use the next planned version of the tested service...

Reply:
i copied the descriptions of cheng from teams